interactions/api/http/sticker.py

```diff
@@ -68,7 +68,7 @@ async def create_guild_sticker(
         :return: The new sticker data on success.
         """
         data = FormData()
-        data.add_field("file", file._fp, filename=file._filename)
+        data.add_field("file", file._fp, filename=file._filename, content_type="image/png")
```

Comment:
Does it work for all types? Also gifs?

Reply:
png and apng works, still figuring out lottie json.
gifs are not supported by discord (apng is the format of animated stickers)